src/Symfony/Bundle/TwigBundle/Resources/config/console.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <defaults public="false" />
+
+        <!-- Alias for BC -->
+        <service id="twig.command.debug" alias="console.command.symfony_bundle_twigbundle_command_debugcommand" />
+        <service id="console.command.symfony_bundle_twigbundle_command_debugcommand" class="Symfony\Bridge\Twig\Command\DebugCommand">
+            <argument type="service" id="twig"/>
+            <tag name="console.command" command="debug:twig" />
+        </service>
```

Comment:
It's done to satisfy [this](https://github.com/symfony/symfony/blob/v3.3.4/src/Symfony/Component/HttpKernel/Bundle/Bundle.php#L188) check.

The deprecated command (in the bundle) will override (by command name) otherwise, as we dont skip deprecated commands here ;-) Note the new service points to the command in the bridge.